### ISSUE TITLE
learning rate when resuming

### ISSUE THREAD

**Issue by adizhol** [sconlyshootery/FeatDepth#12]
Hi,
Thank you for the code!

When I try to resume training from a checkpoint, setting a different learning rate in the cfg file doesn't propagate to the optimizer, and it uses the learning rate of the checkpoint.
Is it on purpose? How could I change it?

**Comment by sconlyshootery** [comment]
Yes, in the resume mode, the learning process will return back to the states of checkpoint, all the setting will be consistent with the input checkpoint.
You can use the finetune mode, just set finetune=<path to a checkpoint> and resume=None in cfg files, the model will only load the weights, other settings like learning rate will follow the new settings.

**Comment by Penitto** [comment]
Hi! Is this feature is only difference between finetune and resume configs?
